index.js

```diff
@@ -41,6 +40,11 @@ async function zeroEks (args) {
   if (visualizeCpuProfile) return cpuProfileVisualization(args)
 
   args.title = args.title || `node ${args.argv.join(' ')}`
+
+  if (checkForTranspiledCode(args.argv[0])) {
```

Comment:
Two more edge cases:
- `args.argv` may be empty if no file path is given:
  ```
  ./cmd.js --
  ```
  In that case I think we should skip the check.
- The file may not exist: it's better if we let the Node.js process start and give an error for that instead of throwing an error from 0x internals that users may not be familiar with
  ```
  ./cmd.js -- node doesnotexist.js
  ```
  So if `readFileSync()` throws an error, we should ignore that and keep going.

Reply:
Check the latest changes and let me know if its good.